storage/src/vespa/storage/distributor/distributor.cpp

```diff
@@ -60,10 +60,11 @@ Distributor::Distributor(DistributorComponentRegister& compReg,
       framework::StatusReporter("distributor", "Distributor"),
       _comp_reg(compReg),
       _metrics(std::make_shared<DistributorMetricSet>()),
+      _total_metrics((num_distributor_stripes == 0) ? std::shared_ptr<DistributorTotalMetrics>() : std::make_shared<DistributorTotalMetrics>(num_distributor_stripes)),
```

Comment:
Consider moving __use_legacy_mode_ above so you can use that instead of _(num_distributor_stripes == 0)_.